src/icon/iconview.js

```diff
@@ -22,36 +22,58 @@ export default class IconView extends View {
 	constructor() {
 		super();
 
-		const bind = this.bindTemplate;
+		/**
+		 * The inline SVG or in legacy mode the name of the icon
+		 * which corresponds with the name of the file in the
+		 * {@link module:theme/iconmanagermodel~IconManagerModel}.
+		 *
+		 * @observable
+		 * @member {String} #content
+		 */
+		this.set( 'content' );
 
 		this.template = new Template( {
 			tag: 'svg',
 			ns: 'http://www.w3.org/2000/svg',
 			attributes: {
-				class: [
-					'ck-icon'
-				]
-			},
-			children: [
-				{
-					tag: 'use',
-					ns: 'http://www.w3.org/2000/svg',
-					attributes: {
-						href: {
-							ns: 'http://www.w3.org/1999/xlink',
-							value: bind.to( 'name', name => `#ck-icon-${ name }` )
-						}
-					}
-				}
-			]
+				class: 'ck-icon',
+				viewBox: '0 0 20 20'
+			}
 		} );
+	}
 
+	init() {
 		/**
-		 * The name of the icon. It corresponds with the name of the
-		 * file in the {@link module:theme/iconmanagermodel~IconManagerModel}.
-		 *
-		 * @observable
-		 * @member {String} #name
+		 * This is a hack for lack of innerHTML binding.
+		 * See: https://github.com/ckeditor/ckeditor5-ui/issues/99.
 		 */
+		return super.init().then( () => {
+			this.on( 'change:content', ( evt, name, value ) => {
```

Comment:
I thought `this.element` is null before init, but is null before setting template definition. I'm moving it to the constructor.